autoware_launch/CHANGELOG.rst

```diff
@@ -2,6 +2,49 @@
 Changelog for package autoware_launch
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.39.0 (2024-11-25)
+-------------------
+* youtalk username
+* Merge commit '9d0e7055a' into release-0.39.0
+* feat: change the old diagnostic_convertor to scenario_simulator_v2_adapter (`#1227 <https://github.com/youtalk/autoware_launch/issues/1227>`_)
```

Comment:
Links in CHANGELOG must be pointing to autowarefoundation

```suggestion
* feat: change the old diagnostic_convertor to scenario_simulator_v2_adapter (`#1227 <https://github.com/autowarefoundation/autoware_launch/issues/1227>`_)
```

Reply:
Oh my gosh. Thank you.

Reply:
@youtalk 

Sorry, I should've mentioned in my previous comment.
You should fix all the other links in your PR as well. Many of them in the auto-generated changelog seems to be pointing to your fork.

Reply:
I'm sorry but fixed. https://github.com/autowarefoundation/autoware_launch/pull/1248/commits/b292c2cff391dc4bfdacefe4bc8e774e21aa2e8e